### PR TITLE
added flexible search test and fixed title/Title typo

### PR DESCRIPTION
**Pull request by robert-e-davidson3**

### CODE REVIEW

File: packages/augur-state/src/Controller.ts [hunk 1]
```diff
@@ -82,11 +82,11 @@ export class Controller<TBigNumber> {
             if (info && info.tags && info.longDescription) {
               this.FTS.add({
                 id: row.id,
-                Title: description,
+                title: description,
```
Comment (brianosaurus) [inline packages/augur-state/src/Controller.ts:85]: oops.

File: packages/augur-state/src/Controller.ts [hunk 2]
```diff
@@ -82,11 +82,11 @@ export class Controller<TBigNumber> {
...
                 description: info.longDescription,
                 tags: info.tags.toString(), // convert to comma separated so it is searchable
                 start: new Date(),
-                end: new Date()
+                end: new Date(),
```
Comment (brianosaurus) [inline packages/augur-state/src/Controller.ts:89]: FYI: There's a PR to add this info to events. #1222

File: packages/augur-test/libs/MakeDbMock.ts [hunk 3]
```diff
@@ -56,18 +60,18 @@ export function makeDbMock() {
...
+      const db = new MockPouchDB(fullDbName, {adapter: "memory"});
+      mockState.dbs[fullDbName] = db;
+      return db;
     };
   }
 
   async function wipeDB(): Promise<void> {
-    await Promise.all(_.map(mockState.dbNames, (dbName) => {
-      const db = new PouchDB(dbName, {adapter: "memory"});
+    await Promise.all(_.map(Object.values(mockState.dbs), (db) => {
```
Comment (brianosaurus) [inline packages/augur-test/libs/MakeDbMock.ts]: I believe TS Arrays have their own map method.

**Review by brianosaurus — APPROVED** [review]
Very nice. 👍

File: packages/augur-test/libs/MakeDbMock.ts (hunk 3 above)
Comment (robert-e-davidson3) [reply packages/augur-test/libs/MakeDbMock.ts]: Awesome!

**Pull request merged**